api/base/urls.py

```diff
@@ -19,8 +20,16 @@
                 url(r'^applications/', include('api.applications.urls', namespace='applications')),
                 url(r'^collections/', include('api.collections.urls', namespace='collections')),
                 url(r'^comments/', include('api.comments.urls', namespace='comments')),
-                url(r'^docs/', include('rest_framework_swagger.urls')),
+                url(r'^nodes/', include('api.nodes.urls', namespace='nodes')),
```

Comment:
DRS does alphabetize on the actual API -> Core api transition, though something weird happens between there and rendering on the documentation page.